Create --preview CLI flag (#2752)
psf/black: docs/the_black_code_style/future_style.md

there is [an experimental option](labels/experimental-string) to enable splitting
strings. We plan to enable this option by default once it is fully stable. This is
tracked in [this issue](https://github.com/psf/black/issues/2188).

## Preview style

> Experimental, potentially disruptive style changes are gathered under the
> `--preview` CLI flag. At the end of each year, these changes may be
> adopted into the default style, as described in
> [The Black Code Style](./index.rst).

Reviewer: ```suggestion
Experimental, potentially disruptive style changes are gathered under the `--preview`
CLI flag. At the end of each year, these changes may be adopted into the default style,
as described in [The Black Code Style](./index.rst).
```

> [The Black Code Style](the_black_code_style/index.rst).

Reviewer: ```suggestion
[The Black Code Style](./index.rst).
```

there is [an experimental option](labels/experimental-string) to enable splitting
strings. We plan to enable this option by default once it is fully stable. This is
tracked in [this issue](https://github.com/psf/black/issues/2188).

## Preview style

Potentially disruptive style changes between major releases are gathered under the

> `--preview` CLI flag. They will eventually become part of main functionality, but until

Reviewer: "will" seems too strong a word; the intention is that the preview features will have time to incubate and become stable. At the end of each year, we'll have to make a call on whether to promote a feature to stable.

Reviewer: Let's do that

> Potentially disruptive style changes between major releases are gathered under the
> `--preview` CLI flag. They will eventually become part of main functionality, but until
> then are described here.

Reviewer: ```suggestion
Experimental, potentially disruptive style changes are gathered under the
`--preview` CLI flag. At the end of each year, these changes may be
adopted into the default style, as described in
[The Black Code Style](the_black_code_style/index.rst).
```
How about this wording
